app/pages/docs/tutorial.mdx

```diff
@@ -97,44 +97,45 @@ my-blitz-app
 └── yarn.lock
 ```
 
-These files are:
+Questi file sono:
 
-- The `app/` folder is a container for most of your project. This is where
-  you’ll put any pages or API routes.
+- La directory `app/` contiene la maggior parte del progetto. Qui è dove
+  verrà inserita ogni pagina o rotta per le API.
```

Comment:
Non tradurrei `route` in questo caso visto che è un termine tecnico.

Reply:
concordo